### PR TITLE
[SEO]: Per page images for social share

### PR DESCRIPTION
**Pull request by khalilcodes**
## Add social share image to other pages
Remaining task from #57 
***

This PR will add support to set per page images for social sharing preview.

### Tasks
- [x] Add frontmatter field `image` and setup in `NextSeo`
- [x] Add docs for it [/docs/seo](https://deploy-preview-101--spectacular-dragon-c1015c.netlify.app/docs/seo)

### CODE REVIEW

File: site/content/docs/seo.md [hunk 1]
```diff
@@ -80,6 +80,20 @@ const config = {
...
+```markdown
+---
+image: /assets/images/your-image-path.jpg
+---
+```
+
 Here is a preview of how it should look when sharing the website link on social sites, for example on Twitter:
 
-<img src="/assets/images/twitter-card.jpg" alt="twitter card" />
+![[twitter-card.jpg]]
```
Comment (rufuspollock) [inline site/content/docs/seo.md:99]: How does that work?

![[twitter-card.jpg]] should become `` in the markup?

File: templates/default/contentlayer.config.js [hunk 2]
```diff
@@ -6,11 +6,11 @@ import wikiLinkPlugin from "remark-wiki-link-plus";
 import remarkMath from "remark-math";
 import rehypeMathjax from 'rehype-mathjax';
 import codeExtra from "remark-code-extra";
-import mermaidPlugin from "mdx-mermaid";
```
Comment (rufuspollock) [inline templates/default/contentlayer.config.js:9]: Why have we removed this?

**Review by rufuspollock — CHANGES_REQUESTED** [review]
Please check the couple of comments.

File: site/content/docs/seo.md (hunk 1 above)
Comment (khalilcodes) [reply site/content/docs/seo.md:99]: ![[twitter-card]] becomes ``

File: templates/default/contentlayer.config.js (hunk 2 above)
Comment (khalilcodes) [reply templates/default/contentlayer.config.js:9]: We are using mdx-mermaid component client side. This import was not being used.

**Pull request merged**